<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaIngreso</class>
 <widget class="QMainWindow" name="VentanaIngreso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="input_Email">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color:#808080;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_Password">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color:#808080;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_iniciarSesion">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>350</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar sesión</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_irRegistrate">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_eresNuevo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>¿Eres nuevo?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_PruebaFoxyBot">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>¡PRUEBA A FOXY BOT!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
